examples/split-screen/layouts/main.json

```diff
@@ -0,0 +1,1020 @@
+{
```

Comment:
Correct me if I am wrong. I don't think Multiple Files for saving won't work for examples. Only single file saving